src/app/account/repo/checks/[id]/form.js

```diff
@@ -19,31 +21,38 @@ export default function Form({ id }) {
 }
 
 export function FormBadge({ src }) {
+  const [copy, setCoppy] = useState(false);
+  const copyHandle = async () => {
+    const url = `![HealthCheck](${src})`;
+    await navigator.clipboard.writeText(url);
+    setCoppy(true);
+  };
   return (
     <div>
-      <label
-        htmlFor="email"
-        className="block text-sm font-medium leading-6 text-white"
-      >
+      <label>
         Add badge to your Repo&lsquo;s README to show the latest check status
       </label>
-      <div className="mt-2 flex rounded-md shadow-sm">
+      <div className="mt-2  rounded-md shadow-sm">
         <div className="relative flex flex-grow items-stretch focus-within:z-10">
-          <div className="pointer-events-none absolute inset-y-0 left-0 flex items-center pl-3">
-            <CheckBadgeIcon
+          <div className=" absolute inset-y-0 left-0 flex items-center pl-3 cursor-pointer">
+            <ClipboardDocumentCheckIcon
               aria-hidden="true"
-              className="h-5 w-5 text-gray-400"
+              className={`h-5 w-5 ${copy ? "text-green-400 " : "text-gray-500 "}  `}
+              onClick={copyHandle}
             />
           </div>
-          <input
+          <Input
             readOnly={true}
             id="badge"
             name="badge"
             type="text"
             value={`![HealthCheck](${src})`}
-            className="block w-full rounded-md border-0 py-1.5 pl-10 text-gray-900 ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+            className="w-full -z-30"
+            inputClassName="pl-10 py-1.5"
+            labelClassName="hidden"
           />
         </div>
+        {copy && <p className=" text-green-400 mt-2">Copied!</p>}
```

Comment:
Extra space not needed here

```suggestion
        {copy && <p className="text-green-400 mt-2">Copied!</p>}
```